api/opds.py

```diff
@@ -430,16 +430,20 @@ def acquisition_links(self, active_license_pool, active_loan, active_hold,
         if can_fulfill:
             if active_loan.fulfillment:
                 # The delivery mechanism for this loan has been
-                # set. There is only one fulfill link.
-                fulfill_links.append(
-                    self.fulfill_link(
-                        data_source_name,
-                        identifier,
-                        active_license_pool,
-                        active_loan,
-                        active_loan.fulfillment.delivery_mechanism
-                    )
-                )
+                # set. There is one link for the delivery mechanism
+                # that was locked in, and links for any streaming
+                # delivery mechanisms.
+                for lpdm in active_license_pool.delivery_mechanisms:
+                    if lpdm is active_loan.fulfillment or lpdm.delivery_mechanism.is_streaming:
+                        fulfill_links.append(
+                            self.fulfill_link(
+                                data_source_name,
+                                identifier,
+                                active_license_pool,
+                                active_loan,
+                                lpdm.delivery_mechanism
+                            )
+                        )
```

Comment:
I think it's probably okay to include that error in the feed, at least at the start. It won't get shown to patrons and it should stop us from tearing our hear out wondering why a book doesn't show up.